vnext/ReactCommon/TEMP_UntilReactCommonUpdate/runtimeexecutor/ReactCommon/RuntimeExecutor.h

```diff
@@ -1,117 +0,0 @@
-/*
```

Comment:
@asklar: Did your compile fix make it into FB's code already? your [PR](https://github.com/facebook/react-native/pull/31110) is still not mreged...
@rectified95: This is in regards to: #7248 There were more changes part of that PR, not sure if they need to be removed...

Reply:
I would not check in until we have confirmation about the  `RuntimEExecutotor.h` 
Maybe FB changed things and we have to update it but still keep the patch.

Reply:
@dannyvv that one is a different PR. The PR for the build break with VS 16.9 is https://github.com/facebook/react-native/pull/31090

Reply:
@dannyvv tl; dr; This is fine.
#7248 only forked RuntimeExecutor.h. An MSBuild  task got added to .vcxproj file to do that neatly - I'm using it for another fork (MapBuffer.cpp), so we can't remove that. The other change in that PR was an unrelated reformatting of a DispatchEvent call, which Alex had introduced in a different check-in.

As for, https://github.com/facebook/react-native/pull/31110 - it was created a day after the latest commit form this batch, and doesn't touch RuntimeExecutor.h